.github/workflows/images.yaml

```diff
@@ -0,0 +1,84 @@
+# NUC image creation and push to DockerHub
+name: NUC Image Push
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push events for the main branch
+  push:
+    branches: [main]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # Build the nuc8i7beh image
+  build_nuc8i7beh:
```

Comment:
Use a build with variables here rather than splitting up like this. All images are the same with a different tag